pagopa/io-app
# chore: [IOBP-523] Add missing `language` parameter in `calculateFees` request

## Short description
This PR adds the missing `language` parameter in `calculateFees` request body

## List of changes proposed in this pull request
- Added `language` parameter in `calculateFees` request, pulled from the redux store.

## How to test
Navigate to Profile > Playgrounds > New Wallet > Payment and start a new payment flow. Check that during the PSP request, the body has the correct `language` parameter.
